README.md

```diff
@@ -20,14 +20,14 @@ Welcoming page displaying brief information about the site, and asking users to
 
 ### Registration
 
-_**/pages/acount/Signup.js**_
+_**/pages/account/Signup.js**_
 Page that consists of a registration form _/components/Form.js_ for new users, asking them to provide the following info:
```

Comment:
It's not a typo. This is specifically how the project structure looks like, so I wouldn't change it. 
Thanks!